mmdet/models/utils/misc.py

```diff
@@ -625,3 +625,28 @@ def aligned_bilinear(tensor: Tensor, factor: int) -> Tensor:
         tensor, pad=(factor // 2, 0, factor // 2, 0), mode='replicate')
 
     return tensor[:, :, :oh - 1, :ow - 1]
+
+
+def unfold_wo_center(x, kernel_size, dilation):
+    """unfold_wo_center, used in original implement in BoxInst:
```

Comment:
unfoled_wo_center -> Unfold without xx center?

Reply:
Yes. without the pixel center.